yarn.lock

```diff
@@ -981,9 +981,23 @@
   resolved "https://registry.yarnpkg.com/@types/q/-/q-1.5.4.tgz#15925414e0ad2cd765bfef58842f7e26a7accb24"
   integrity sha512-1HcDas8SEj4z1Wc696tH56G8OlRaH/sqZOynNNB+HF0WOeXPaxTtbYzJY2oEfiUxjSKjhCKr+MvR7dCHcEelug==
 
-"@uiowa/uids@uiowa/uids#3.1.6":
-  version "3.1.6"
-  resolved "https://codeload.github.com/uiowa/uids/tar.gz/40c22b759b59135cc5476c9cb2940ab0a599dbad"
+"@uiowa/uids@uiowa/uids":
+  version "3.1.7"
+  resolved "https://codeload.github.com/uiowa/uids/tar.gz/b0496aa6ba74793ed0c1907b6df336bd57b5c342"
+  dependencies:
+    autoprefixer "^9.8.0"
+    cssnano "^4.1.10"
+    del "^5.1.0"
+    gulp "^4.0.0"
+    gulp-imagemin "^7.1.0"
+    gulp-postcss "^8.0.0"
+    gulp-sass "^4.1.0"
+    gulp-sass-glob "^1.0.9"
+    gulp-sourcemaps "^2.6.4"
+
+"@uiowa/uids@uiowa/uids#3.1.8":
+  version "3.1.8"
+  resolved "https://codeload.github.com/uiowa/uids/tar.gz/1325323bd4634f3646472976be26196fd0a33460"
```

Comment:
Is this a quirk of a new version of yarn? It seem like we shouldn't have multiple definitions for UIDS in here.

Reply:
I removed the extra entry and it didn't seem to cause any issues.